packages/repositories.config

```diff
@@ -1,18 +1,7 @@
 ﻿<?xml version="1.0" encoding="utf-8"?>
 <repositories>
-  <repository path="../src/NUnitEngine/Addins/nunit.v2.driver.tests/packages.config" />
-  <repository path="../src/NUnitEngine/nunit.engine.tests/packages.config" />
-  <repository path="../src/NUnitEngine/nunit.engine/packages.config" />
   <repository path="../src/NUnitFramework/testdata/packages.config" />
   <repository path="../src/NUnitFramework/tests/packages.config" />
-  <repository path="..\src\NUnitEngine\Addins\nunit.v2.driver.tests\packages.config" />
-  <repository path="..\src\NUnitEngine\mock-v2-assembly\packages.config" />
-  <repository path="..\src\NUnitEngine\nunit.core.engine\packages.config" />
-  <repository path="..\src\NUnitEngine\nunit.engine.tests\packages.config" />
-  <repository path="..\src\NUnitEngine\nunit.engine\packages.config" />
-  <repository path="..\src\NUnitEngine\nunit.v2.driver.tests\packages.config" />
   <repository path="..\src\NUnitFramework\testdata\packages.config" />
   <repository path="..\src\NUnitFramework\tests\packages.config" />
```

Comment:
Odd, is this supposed to be repeated, but with inverted `/` ? seems a bit odd to me never noticed this before.